### PR TITLE
[W5.4][W15-A1] Lim Shunyong

### PR DESCRIPTION
**Pull request by limshunyong**
Enhancement for Addressbook- level 3
1.Created Sort Command 
2.Update user guide
3. update JUnit testing

### CODE REVIEW

File: src/seedu/addressbook/commands/SortCommand.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package seedu.addressbook.commands;
+
+
+/**
+ * Sort the address book based on the name of the contact.
```
Comment (okkhoy) [inline src/seedu/addressbook/commands/SortCommand.java:5]: *Sorts* the address ...

File: src/seedu/addressbook/data/person/UniquePersonList.java [hunk 2]
```diff
@@ -131,4 +131,14 @@ public int hashCode() {
         return internalList.hashCode();
     }
 
+
+    /**
+     * Sort the list
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/UniquePersonList.java:136]: *Sorts* the list

File: src/seedu/addressbook/data/person/UniquePersonList.java [hunk 3]
```diff
@@ -131,4 +131,14 @@ public int hashCode() {
...
+     */
+
+    public void sort() {
+        internalList.sort((person1, person2) ->
+                (person1.getName().fullName.
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/UniquePersonList.java:141]: :+1:

File: test/java/seedu/addressbook/logic/LogicTest.java [hunk 4]
```diff
@@ -457,6 +457,45 @@ public void execute_find_matchesIfAnyKeywordPresent() throws Exception {
                                 expectedList);
     }
 
+    @Test
+    public void execute_sort_command() throws Exception {
```
Comment (okkhoy) [inline test/java/seedu/addressbook/logic/LogicTest.java:461]: :+1: good test case
going further please check how to write test names, for now this is OK

**Comment by okkhoy** [comment]
@limshunyong 
Some comments added. Please ack & close the PR after reading comments.

**Comment by limshunyong** [comment]
@okkhoy , comments acknowledged. Thank you.

**Pull request closed without merging**